docs/apm/agent-configuration.asciidoc

```diff
@@ -0,0 +1,28 @@
+[role="xpack"]
+[[agent-configuration]]
+=== APM Agent configuration
+
+beta[] APM Agent configuration allows you to fine-tune your agent configuration directly in Kibana.
+Best of all, changes are automatically propagated to your APM agents so there's no need to redeploy.
+
+To get started, simply choose the service and environment you wish to configure.
+Agents will periodically poll APM Server and apply any configuration changes.
+APM Agent configuration takes precedence over any settings set locally in each agent.
+
+[role="screenshot"]
+image::apm/images/apm-agent-configuration.png[APM Agent configuration in Kibana]
+
+IMPORTANT: As this feature is in Beta, a limited number of configuration settings are supported.
+We will extend support in future releases.
+If you have feedback, please reach out in our https://discuss.elastic.co/c/apm[Discuss forum].
+
+==== Supported configurations
+
+===== `TRANSACTION_SAMPLE_RATE`
+
+added[7.3.0] A sample rate between `0.00` and `1.0`. Default configuration is `1.0` (100% of traces).
```

Comment:
I think we changed this to `0.000` because we want users to be able to do 0,1% sampling. I will change the help texts in the UI.

Reply:
Thanks!